apps/anon-message-client/src/app/[nullifier]/page.tsx

```diff
@@ -0,0 +1,79 @@
+import Post from "@/components/post";
+import { bigIntToPseudonymEmoji, bigIntToPseudonymName } from "@pcd/util";
+import { Suspense } from "react";
+import Loading from "./loading";
+
+interface AnonMessageWithDetails {
+  id: number;
+  nullifier: string;
+  chat_topic_id: number;
+  content: string;
+  proof: string;
+  chat_name: string;
+  topic_name: string;
+  message_timestamp: string;
+}
+
+export default async function Page({
+  params
+}: {
+  params: { nullifier: string };
+}) {
+  const getData = async (): Promise<AnonMessageWithDetails[]> => {
+    const data = await fetch(
+      `${process.env.NEXT_PUBLIC_PASSPORT_SERVER_URL}/telegram/anonget/${params.nullifier}`,
+      { cache: "no-store" }
+    );
+    const dataJson = await data.json();
+    return dataJson;
+  };
+
+  const messages = await getData();
+
+  return (
+    <Suspense fallback={<Loading />}>
+      <div className="flex flex-col items-center bg-white p-4">
```

Comment:
could we add a "copy link" somewhere in the page?

Reply:
#1148